nservicebus/hosting/expose-an-endpoint-as-a-web-wcf-service.md

```diff
@@ -10,6 +10,8 @@ related:
  - samples/web/wcf-callbacks
 ---
 
+Note: NServiceBus Version 6 and above no longer provide built-in support for WCF integration. Refer to the [Upgrade Guide](/nservicebus/upgrades/host-6to7) for further information.
```

Comment:
I'm pretty sure we used NOTE for deprecation notice in other places of the docs. :+1: on the suggested text though